frontend/src/old-pages/CustomImages/CustomImageDetails.tsx

```diff
@@ -38,6 +39,10 @@ import {ValueWithLabel} from '../../components/ValueWithLabel'
 // Constants
 const customImagesPath = ['app', 'customImages']
 
+function truncate(string: string, limit: number) {
```

Comment:
todo: the same util is already built in [lodash](https://lodash.com/docs/4.17.15#truncate)